lib/didomi_sdk.dart

```diff
@@ -51,7 +51,7 @@ class DidomiSdk {
   static Future<bool> get isReady async => await _channel.invokeMethod('isReady');
 
   /// Check if user consent are partial or not set
-  @Deprecated("Use shouldUserStatusBeCollected instead.")
+  @Deprecated('Use ‘shouldUserStatusBeCollected‘ instead.')
```

Comment:
Is there any reason to move back to simple quote? Double quotes often offer more advantage...

Reply:
Not really, i just wanted to test if it made a difference when displayed in the IDE (it does not)